### ISSUE TITLE
non local参数

### ISSUE THREAD

**Issue by bixiaopeng0** [JDAI-CV/fast-reid#256]
## ❓ How to do something using fastreid

`class Non_local(nn.Module):
    def __init__(self, in_channels, bn_norm, num_splits, reduc_ratio=2):
        super(Non_local, self).__init__()

        self.in_channels = in_channels
        self.inter_channels = reduc_ratio // reduc_ratio`

请问bn_norm，num_splits和reduc_ratio这几个参数一般怎么设置啊

NOTE:

1. Only general answers are provided.
   If you want to ask about "why X did not work", please use the
   [Unexpected behaviors](https://github.com/JDAI-CV/fast-reid/issues/new/choose) issue template.

2. About how to implement new models / new dataloader / new training logic, etc., check documentation first.

3. We do not answer general machine learning / computer vision questions that are not specific to fastreid, such as how a model works, how to improve your training/make it converge, or what algorithm/methods can be used to achieve X.

**Comment by L1aoXingyu** [comment]
`bn_norm` 表示用哪种 normalization，比如可以用 instance norm，也可以用 batch norm 等等，我们默认是用 batch norm；`num_splits` 这个参数现在已经去掉了，这个参数主要是配合 GhostBN 一起使用的，对于其他 normalization 没有作用；`reduc_ratio` 主要是中间的 feature dim 和 in channel 的比例，你可以去看看原始论文

**Comment by Hzj199** [comment]
> `bn_norm` 表示用哪种 normalization，比如可以用 instance norm，也可以用 batch norm 等等，我们默认是用 batch norm；`num_splits` 这个参数现在已经去掉了，这个参数主要是配合 GhostBN 一起使用的，对于其他 normalization 没有作用；`reduc_ratio` 主要是中间的 feature dim 和 in channel 的比例，你可以去看看原始论文

self.inter_channels = reduc_ratio // reduc_ratio 这里是不是有问题，self.inter_channels = 1
